exports[`test throwing not Error objects 1`] = `
Object {
  "rest": " FAIL  __tests__/throw-number-test.js
  ● Test suite failed to run

    Expected an Error, but \"1\" was thrown
    


",
  "summary": "Test Summary: › Ran all tests matching \"throw-number-test.js\".

suites:    1 failed, 0 passed (1 total)
tests:     0 passed (0 total)
snapshots: 0 passed (0 total)
time:      <<REPLACED>>
"
}
`;

exports[`test throwing not Error objects 2`] = `
Object {
  "rest": " FAIL  __tests__/throw-string-test.js
  ● Test suite failed to run

    Error
      banana


",
  "summary": "Test Summary: › Ran all tests matching \"throw-string-test.js\".

suites:    1 failed, 0 passed (1 total)
tests:     0 passed (0 total)
snapshots: 0 passed (0 total)
time:      <<REPLACED>>
"
}
`;

exports[`test throwing not Error objects 3`] = `
Object {
  "rest": " FAIL  __tests__/throw-object-test.js
  ● Test suite failed to run

    Error: No message was provided


",
  "summary": "Test Summary: › Ran all tests matching \"throw-object-test.js\".

suites:    1 failed, 0 passed (1 total)
tests:     0 passed (0 total)
snapshots: 0 passed (0 total)
time:      <<REPLACED>>
"
}
`;
